ontoma/__init__.py

```diff
@@ -1,20 +1,12 @@
+__all__ = ['OnToma']
 
-# -*- coding: utf-8 -*-
+import logging
 
-__all__ = [
-    'OnToma'
-]
 from ontoma.interface import OnToma
 
-
-import logging
-
 logger = logging.getLogger(__name__)
```

Comment:
True, logging in OnToma needs an overhaul. The way it's currently set up is also probably the reason for getting duplicate lines in the logs.

I want to have a proper solution for this, which will require a bit of investigation into the best practices for the `logging` module. So I also agree that it's best handled as a follow-up issue rather than a blocker. Added a point to this follow-up ticket: https://github.com/opentargets/platform/issues/1619